kiva/tests/test_svg_drawing.py

```diff
@@ -26,8 +26,11 @@ def draw_and_check(self):
         self.gc.save(filename)
         tree = ElementTree.parse(filename)
         elements = [element for element in tree.iter()]
-        if not len(elements) in [4, 7]:
-            self.fail("The expected number of elements was not found")
+        if not len(elements) in [4, 5, 7]:
```

Comment:
Might as well change the unidiomatic `if not ... in` to the more idiomatic `if ... not in` while we're touching this code.